packages/dotnet-sdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+Welcome and thank you for your interest in contributing to **Microsoft.TeamsFx.Blazor**! Before contributing to this project, please review this document for policies and procedures which will ease the contribution and review process for everyone. If you have questions, please raise your issue on github.
```

Comment:
Fix package name and remove "Blazor" term

Reply:
Thank you John! I missed this change.